UnitsNet.Tests/CustomCode/MolarFlowTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+using UnitsNet.Units;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class MolarFlowTests : MolarFlowTestsBase
+    {
+        // Override properties in base class here
+        protected override bool SupportsSIUnitSystem => false;
+
+        protected override double KilomolesPerHourInOneKilomolePerSecond => 3.599997e3;
+
+        protected override double KilomolesPerMinuteInOneKilomolePerSecond => 5.999988e1;
+
+        protected override double KilomolesPerSecondInOneKilomolePerSecond => 1.0;
+
+        protected override double PoundsMolePerHourInOneKilomolePerSecond => 7.93663391482404e3;
```

Comment:
I found: 7936.64143866

https://www.convert-measurement-units.com/convert+Kilomole+to+Pound-mole.php
then multiplying by 3600
https://www.google.com/search?q=2.2046226218488+*+3600